packages/react-divider/src/components/Divider/Divider.types.ts

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
```

Comment:
Same comments here as I made in the API section of the spec.

Reply:
Sorry I'm not sure .. can you clarify? Is this outdated?